### PR TITLE
Some small fixes

### PR DESCRIPTION
**Pull request by LourensVeen**
While working on #61, I ran into some small issues, which are collected here in a separate PR in an attempt to cut the whole thing into reviewable pieces.

### CODE REVIEW

File: proof_of_concept/rest/internal_client.py [hunk 1]
```diff
@@ -78,11 +83,30 @@ def submit_job(self, job: Job) -> str:
             raise RuntimeError(f'Error submitting job: {r.text}')
         if 'location' not in r.headers:
             raise RuntimeError('Invalid server response when submitting job')
+
+        # Protect against malicious servers redirecting us elsewhere
+        std_ports = {'http': 80, 'https': 443}
```
Comment (svenvanderburg) [inline proof_of_concept/rest/internal_client.py]: Factor out a constant maybe?

**Review by svenvanderburg — APPROVED** [review]
Looks good! I just added 1 annoying comment...

File: proof_of_concept/rest/internal_client.py (hunk 1 above)
Comment (LourensVeen) [reply proof_of_concept/rest/internal_client.py]: Haha :-). How about this:
```
HTTP_SCHEMA = 'http'
HTTP_PORT = 80
HTTPS_SCHEMA = 'https'
HTTPS_PORT = 443
STD_PORTS = {HTTP_SCHEMA: HTTP_PORT, HTTPS_SCHEMA: HTTPS_PORT}
```

Comment (svenvanderburg) [reply proof_of_concept/rest/internal_client.py]: That is maybe too much?

For me
```python
STANDARD_PORTS = {'http': 80, 'https': 443}
```
would be good enough.

Would the overall software quality of the project really be affected by what you choose here? Probably not... 🦊

Comment (LourensVeen) [reply proof_of_concept/rest/internal_client.py]: NEEEEDS_MOOOOOOOR_CONSTANTS!

You know, I could make all the strings in my error messages constants...Hmmmm.

**Pull request merged**